.github/workflows/cd-linux.yaml

```
@@ -58,8 +62,8 @@ jobs:
             libatk1.0-dev \
             libpangocairo-1.0-0 \
             libpango1.0-dev \
-            make \
-            python3-poetry
+            make
+          pip install poetry
```

Sorry, I missed this:

```suggestion
          pip install "${POETRY_SPEC}"
```